testdata/dnn/download_models.py

```diff
@@ -952,6 +952,26 @@ def get_confirm_token(response):  # in case of large files
         url='https://drive.google.com/uc?export=dowload&id=1ClK9WiB492c5OZFKveF3XiHCejoOxINW',
         sha='12ff8b1f5c8bff62e8dd91eabdacdfc998be255e',
         filename='onnx/models/face_recognizer_fast.onnx'),
+    Model(
+        name='wechat_qr_detect',
+        url='https://raw.githubusercontent.com/WeChatCV/opencv_3rdparty/wechat_qrcode/detect.prototxt',
+        sha='a6936962139282d300ebbf15a54c2aa94b144bb7',
+        filename='wechat/detect.prototxt'),
```

Comment:
`wechat` => `wechat-YYYYMM` to support versioning (because download shared directory doesn't support versioning)

Reply:
Notes are linked to the last line.

Reply:
@alalek, I added prefix `wechat_2021-01` to path. Should you load model? Then I'll restart the tests

Reply:
Downloaded

Reply:
@alalek, 
[Ubuntu2004-x64:](https://github.com/opencv/opencv_contrib/runs/7841254690?check_suite_focus=true#logs)
`following models have not been downloaded: wechat_2021-01/detect.prototxt`


default build:

`C++ exception with description "OpenCV(4.6.0-dev) /build/precommit-contrib_linux64/4.x/opencv/modules/ts/src/ts.cpp:1064: error: (-2:Unspecified error) OpenCV tests: Can't find required data file: ../dnn/wechat_2021-01/detect.prototxt in function 'findData'`

Reply:
others CI are ok